drivers/gpu/drm/vc4/vc4_firmware_kms.c

```
@@ -683,6 +683,7 @@ static struct drm_plane *vc4_fkms_plane_init(struct drm_device *dev,
 	 * other layers as requested by KMS.
 	 */
 	switch (type) {
+	default:
```

Is the fallthrough intentional here? Or should we return an error in default and have a `break`? Either way, I guess a fallthrough comment is needed...

---

good point;
i think i should also try to update the GCC version to catch some of these things;